### PR TITLE
Test/acceptance test burst shot #61

### PR DESCRIPTION
**Pull request by aa-software2112**

### CODE REVIEW

File: app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        Thread.sleep(SHUTTER_TO_BURST_MODE_TIME)
+
+        /** Verifying button visibilities in burst shot mode */
+        onView(anyOf(withId(R.id.shutter))).check(matches(not(isDisplayed())));
+        onView(anyOf(withId(R.id.burst))).check(matches(isDisplayed()));
+
+        /** Release the button */
+        onView(withId(R.id.shutter)).perform(ViewActionHelper.release())
+
+        var finalNumPhotos = getNumPhotos()
```
Comment (aa-software2112) [inline app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt]: remove this

File: app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+        /** Verifying button visibilities after leaving burst shot mode */
+        onView(anyOf(withId(R.id.burst))).check(matches(not(isDisplayed())));
+        onView(anyOf(withId(R.id.shutter))).check(matches(isDisplayed()));
+
+        assertTrue(finalNumPhotos > 1)
```
Comment (aa-software2112) [inline app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt]: remove this

File: app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        /** Release the button */
+        //onView(withId(R.id.shutter)).perform(ViewActionHelper.release())
+    }
+
+    /** Acceptance Test 7: Functional: Continuous captures on shutter button hold when not in countdown mode and not in video mode: Automatic
+     * 1. Application Starts
+     * 2. Hold down shutter for 3 seconds - 2seconds to activate the burst
+     * 3. Verifying camera is neither in countdown mode nor video mode
+     * 4. Verifying that shutter button is INVISIBLE and that burst button is VISIBLE in this state
+     * 5. Photos are taken in burst mode for 1 second
```
Comment (aa-software2112) [inline app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt]: Remove step 5

File: app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+     * 6. Burst button is released
+     * 7. Verifying that shutter button is VISIBLE and that burst button is INVISIBLE in this state
+     * 8. Verifying that more than one photo was taken
```
Comment (aa-software2112) [inline app/src/androidTest/kotlin/com/simplemobiletools/camera/activities/BurstShotTest.kt]: Remove step 8

**Review by aa-software2112 — COMMENTED** [review]
@laurawheatley check the comments

**Comment by lkadian** [comment]
Looks good. I'll run ktlint and merge.

**Pull request merged**